doc/classes/Rect2i.xml

```diff
@@ -70,41 +82,51 @@
 			<return type="Rect2i" />
 			<param index="0" name="to" type="Vector2i" />
 			<description>
-				Returns a copy of this [Rect2i] expanded so that the borders align with the given point.
+				Expands this rectangle to contain the given [param to] point inside it, if necessary.
```

Comment:
This description is ambiguous, because users might interpret "contain" as a synonym for the function `has_point`, but `r.expand(p).has_point(p) == false`, if `p` is right or below of `r`. A similar argument can be made for `Rect2.expand`.

Reply:
You may check again if you wish

Reply:
Thanks for taking this into consideration!